### PR TITLE
Parallel Run, remove prod-dev clustser in cmbackstage and cleanup

### PR DESCRIPTION
**Pull request by hmuthusamy**
*Issue #, if available:*

*Description of changes:*

1. Removed README from database folder
2. Removed dev and prod cluster reference in the CM config for backstage
3. Added Wait and parallel terraform apply to reduce the time

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/backstage/dev/cm-backstage-config.yaml [hunk 1]
```diff
@@ -135,13 +135,5 @@ data:
               url: http://${DOMAIN_NAME}/argocd
               username: admin
               password: ${ARGOCD_ADMIN_PASSWORD}
-            - name: dev-cluster
```
Comment (shapirov103) [inline packages/backstage/dev/cm-backstage-config.yaml:138]: Why is this not necessary anymore after this change?

**Review by shapirov103 — COMMENTED** [review]
@hmuthusamy LGTM, please see my small question. 

Was it tested? (assume it was)

**Comment by hmuthusamy** [comment]
> @hmuthusamy LGTM, please see my small question.
> 
> Was it tested? (assume it was)

Yes. Tested multiple times locally destroyed and recreated. Also re-run was quicker.

File: packages/backstage/dev/cm-backstage-config.yaml (hunk 1 above)
Comment (hmuthusamy) [reply packages/backstage/dev/cm-backstage-config.yaml:138]: We are deploying the templates only backstage in mgmt cluster. DEV and PROD cluster dont have argocd installed. So when deploying templates you see 3 iterations instead of one right one. This will fix that issue

Comment (shapirov103) [reply packages/backstage/dev/cm-backstage-config.yaml:138]: makes sense

File: platform/infra/terraform/database/README.md [hunk 2]
```diff
@@ -1,122 +0,0 @@
-# Terraform AWS Infrastructure Project
-
```
Comment (elamaran11) [inline platform/infra/terraform/database/README.md]: Why is this file removed. I think it can exist without harm. Its to say how the script works.

**Review by elamaran11 — COMMENTED** [review]
One Minor comment.

File: platform/infra/terraform/database/README.md (hunk 2 above)
Comment (hmuthusamy) [reply platform/infra/terraform/database/README.md]: @elamaran11 Added it back

**Review by elamaran11 — APPROVED** [review]
LGTM
